src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -94,14 +94,35 @@
 					:conversation-name="conversationNameTrimmed" />
 
 				<!-- Third page -->
-				<Confirmation v-else-if="page === 2"
-					class="new-group-conversation__content"
-					:token="newConversation.token"
-					:conversation-name="conversationNameTrimmed"
-					:error="error"
-					:is-loading="isLoading"
-					:success="success"
-					:is-public="isPublic" />
+				<div v-else-if="page === 2" class="confirmation new-group-conversation__content">
```

Comment:
```suggestion
				<div v-else-if="page === 2" class="new-group-conversation__content">
```

We could omit that class, as it doesn't have any styles applied